test/fixtures/full_config.yml

```diff
@@ -2309,8 +2321,16 @@ Naming/MethodParameterName:
   - pp
   - to
   ForbiddenNames: []
-Naming/PredicateName:
-  Description: Check the names of predicate methods.
+Naming/PredicateMethod:
+  Description: Checks that predicate methods end with `?` and non-predicate methods
+    do not.
+  Enabled: true
```

Comment:
Let's not enable this one. No cop in the Naming category was enabled just yet. This one specifically can force things that I don't think we want.